test/functional/wallet.py

```diff
@@ -93,11 +93,17 @@ def run_test(self):
         # Exercise locking of unspent outputs
         unspent_0 = self.nodes[2].listunspent()[0]
         unspent_0 = {"txid": unspent_0["txid"], "vout": unspent_0["vout"]}
+        assert_raises_jsonrpc(-8, "expected locked output", self.nodes[2].lockunspent, True, [unspent_0])
```

Comment:
Personally, I don't think the complete error message needs to be asserted, as long as the string is long enough to disambiguate. But also fine to change this to the complete string.